.releaserc

```diff
@@ -1,5 +1,10 @@
 {
-  "branches": ["master", "next"],
+  "branches": [
+    "master",
+    "next",
+    { "name": "alpha", "prerelease": true },
+    { "name": "beta", "prerelease": true }
+  ],
```

Comment:
Before this PR lands this file should show no changes. This will mean making a PR to current `master` updating this file to deploy from `master` and `22.x`.